### ISSUE TITLE
spells casting time should be standardized

### ISSUE THREAD

**Issue by TSPARR** [ZachMyers3/flask-dnd#3]

**Comment by ZachMyers3** [comment]
The only edge cases that are an issue currently are Bonus Action and Action, which can both represent the same unit of time (8 seconds).

I suggest making casting_time an object similar to materials and add an action_type field, example below.

```json
  {
    "name": "Misty Step",
    "desc": "Briefly surrounded by silvery mist, you teleport up to 30 feet to an unoccupied space that you can see.",
    "book": "Player's Handbook",
    "casting_time_before": "Bonus Action",
    "casting_time_after": {
        "action_type": "bonus",
        "duration": "8 seconds"
  }
```

**Comment by ZachMyers3** [comment]
Fixed in [fc7300c](https://github.com/ZachMyers3/flask-dnd/commit/fc7300c36b02e0c121a0c133df729b2a9ce31cbd)
